tools/gen_compilation_database.py

```diff
@@ -111,6 +115,7 @@ def fix_compilation_database(args, db):
     parser.add_argument('--include_headers', action='store_true')
     parser.add_argument('--vscode', action='store_true')
     parser.add_argument('--include_all', action='store_true')
+    parser.add_argument('--include_contrib', action='store_true')
```

Comment:
Let's default this to true, otherwise it will break clang-tidy in contrib PRs.

Reply:
the easiest way to do that it is to flip the switch - eg `--exclude_contrib`

Reply:
Changed to `--exclude_contrib`